.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: Unit Tests
+
+on:
+  push:
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+# Cancels all previous workflow runs for the same branch that have not yet completed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: ['7.3', '7.4', '8.0', '8.1']
+        composer-flags: ['']
+
+        include:
+          - php: '8.2'
+            composer-flags: '--ignore-platform-reqs'
+
+    name: PHP ${{ matrix.php }}
+
+    continue-on-error: ${{ matrix.php == '8.2' }}
```

Comment:
`continue-on-error` is *not* an equivalent of the `allow_failures` of Travis. It is about running next steps even if a previous step fails.

Reply:
I'm aware, but it is the next best thing until https://github.com/actions/toolkit/issues/399 has been resolved.

Reply:
I'd rather not have the PHP 8.2 job yet then. We'll add it when we're closer to the PHP 8.2 release (anyway, neither Prophecy nor PHPUnit are advocating support for it yet anyway)

Reply:
Fair enough. I've removed the `8.2` build, as well as the `composer_options` flag in the matrix as it is no longer needed without the `8.2` build.